src/main/plugin/iso19139.ca.HNAP/loc/eng/labels.xml

```diff
@@ -387,7 +387,7 @@
     <element name="gmd:MD_ReferenceSystem" id="186.0">
         <label>Reference system</label>
         <description>Information about the reference system.</description>
-        <help>information about the reference system</help>
+        <!--<help>information about the reference system</help>-->
```

Comment:
It may be cleaner to just remove these

Reply:
> It may be cleaner to just remove these

I totally agree. I was just worrying if other people will object to "removing" it because they want to keep some information. I will wait a little bit and see if there is no other objection, then I will commit another version to have all of them removed.

Reply:
@wangf1122  - I also recommend that you just remove the text.  If ever we need it again - it can be retrieved from git

Reply:
Another version is committed. All comments are removed.